static/js/muting_ui.js

```diff
@@ -16,6 +16,46 @@ exports.rerender = function () {
     }
 };
 
+exports.mute_notification = (function () {
+    var timeout,
+        animate = {
+            fadeOut: function ($mute) {
+                $mute.fadeOut(500).removeClass("show");
+            },
+            fadeIn: function ($mute) {
+                $mute.fadeIn(500).addClass("show");
+                clearTimeout(timeout);
+            }
+        },
+        event_added = false;
+
+    return function (stream, topic) {
+        var $mute_topic = $("#mute_topic_container"),
```

Comment:
I would consider renaming the id to something like '#undo_mute_topic_container," so people are clear this is part of the little undo widget.

Then I would definitely rename $mute_topic, which most readers will assume is a topic name and not a UI container.  Since you're in muting_ui.js, you could keep the name concise by using something like $undo_container.